src/modules/cvrp/processor/vrp_processor.py

```diff
@@ -411,8 +428,8 @@ def map_slo_cro_deliveries(self, deliveries, event_type):
         """
         delivery_parts = [[],[]] # First one is for SLO nodes, Second one is for CRO nodes
         for parcel in deliveries:
-            if "S" in parcel.current_location or "PS" in parcel.current_location:
-                if "H" in parcel.target or "HP" in parcel.target:
+            if "SLO" == parcel.country:
```

Comment:
the same here and on the other places where "CRO" and "SLO" apeared

Reply:
fixed